.github/workflows/generate-analyzer-docs.yml

```diff
@@ -0,0 +1,33 @@
+name: Generate Analyzer docs
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        required: true
+        description: "Documentation version to generate"
+      source-ref:
```

Comment:
at first I was going to suggest that we default to `version`, but if we're running this manually, it's probably because we want them to be different!